sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/ViewBuilder.java

```diff
@@ -6,19 +6,39 @@
 package io.opentelemetry.sdk.metrics.view;
 
 import io.opentelemetry.sdk.metrics.aggregator.AggregatorFactory;
+import io.opentelemetry.sdk.metrics.processor.LabelsProcessorFactory;
 import java.util.Objects;
 
 public final class ViewBuilder {
   private AggregatorFactory aggregatorFactory;
+  private LabelsProcessorFactory labelsProcessorFactory;
```

Comment:
it'll be a little more idiomatic with the rest of our builders if you assign the default value (noop) here, rather than in the `View.builder()` call.

Reply:
done